server/graphql/v2/query/collection/AccountsQuery.ts

```diff
@@ -52,8 +53,9 @@ const AccountsQuery = {
       description: 'Limit the search to collectives belonging to these countries',
     },
     orderBy: {
-      type: OrderByInput,
+      type: new GraphQLNonNull(OrderByInput),
       description: 'The order of results',
+      defaultValue: { field: 'RANK', direction: 'DESC' },
```

Comment:
@Betree : I've defaulted this to `RANK` for all cases for now as the `ACTIVITY` sort is not efficient when there's no search term. I believe after, https://github.com/opencollective/opencollective/issues/5454 we should change this default field to `ACTIVITY`.

Reply:
@SudharakaP This will still crash:

![image](https://user-images.githubusercontent.com/1556356/167574609-52f964a5-c3fd-40ea-a657-bfde99765fb8.png)

As per my comment above, I'm going with a different fix in a separate PR. Can we keep this one only about updating the `deprecationReason`?

Reply:
Ah, I see the problem now, I was thinking that we were using the `RANK` in the same way in both branches. Anyways, yes I've update the PR to keep it only for `deprecationReason`.